utils/appliance/__init__.py

```diff
@@ -1694,7 +1694,7 @@ def server_roles(self, roles):
         yaml = self.get_yaml_config()
         yaml['server']['role'] = ','.join([role for role, boolean in roles.iteritems() if boolean])
```

Comment:
based on this line the given description is a lie as the roles arent merged/joined

thecheck is just worse

if the new code works by a hidden interaction, then it deserves a propper code lineup, else this is simply incomprehensible after just a few days